Source/WebKit/WebProcess/Extensions/API/WebExtensionAPIScripting.h

```diff
@@ -76,6 +75,8 @@ NSArray *toWebAPI(const Vector<WebExtensionScriptInjectionResultParameters>&, bo
 NSArray *toWebAPI(const Vector<WebExtensionRegisteredScriptParameters>&);
 NSString *toWebAPI(WebExtension::InjectionTime);
 
+void parseRegisteredContentScripts(NSArray *, FirstTimeRegistration, Vector<WebExtensionRegisteredScriptParameters>&);
```

Comment:
This can be a static on the class, so it does not need to move out.

`static void parseRegisteredContentScripts(NSArray *, FirstTimeRegistration, Vector<WebExtensionRegisteredScriptParameters>&);` in the `class` above, like it was before. Might need to make it `public` or add `friend class Foo` for the class that needs to access it. `WebExtensionAPIAction` does this for `WebExtensionAPIMenus`.